v22.2/cluster-setup-troubleshooting.md

```diff
@@ -304,6 +304,30 @@ The solution is to add connection retry logic to your application.
 
 ## Clock sync issues
 
+#### A node's timezone data has changed
+
+CockroachDB relies on each node's operating system timezone database to resolve named timezones like `America/New_York`, and for applying daylight saving rules. This can lead to inconsistent results across a cluster when coercing date/time strings into temporal types.
+
+Ensure that each node's operating system uses the same timezone database. In general, Cockroach Labs recommends that each node has an identical hardware and software configuration.
+
+Timezone data is read from the following sources in order of preference:
+
+1. The `ZONEINFO` operating system environment variable, if it is set to point to a `zoneinfo.zip` file such as the one that is included in Go.
```

Comment:
ZONEINFO can point either to a zip filr or a directory containing the tzdata. This can thus also be used e.g. on OS variants what have their tzdata in non-standard locations.